docs/manuals/source/DeveloperGuide/ReleasingBareos.rst

```diff
@@ -0,0 +1,77 @@
+Releasing Bareos
+================
+
+This chapter describes how to release a new version of Bareos.
+The documentation is limited to the changes that need to be done to the sources and the git repository.
+Building and distributing source and binary packages is not the scope of this chapter.
+
+.. note::
+
+   Consider this a step-by-step manual. The sections are meant to be carried out in that order.
+
+Deciding for a version number
+-----------------------------
+When you release you will usually need at least two version numbers: the version you're going to release and the version that will follow after that version.
+For details on the numbering-scheme, take a look at :ref:`section-version-numbers`.
+
+Usually, you will have an existing work-in-progress tag that generates pre-release version numbers for your branch.
+So the version you're going to release is probably the final version for that pre-release.
+
+Preparing the release notes
+---------------------------
+For each new release there should be a section in the :ref:`releasenotes`.
+You can use the release notes of a previous release as a template.
+
+The release notes should be committed to the master branch and then cherry-picked to your release branch using ``git cherry-pick -x``.
+
+Update version-dependent files
+------------------------------
+There are version-dependent files in the Bareos sources that might need your attention.
+
+version.map.in
+~~~~~~~~~~~~~~
+The file is in either :file:`core/src/cats/ddl` or :file:`src/cats/ddl`.
+Whenever the database schema version changes this file must be updated.
+Usually the developer who did a schema change should have done this.
+However, please double-check and add or update the version mapping if needed.
+
+Prepare the git commits and tags
+--------------------------------
+There is a helper script :file:`prep-release.sh` that will help with the process.
+You can just call the script and it will handle the version-dependant things that need to happen for the release.
+The script will also prepare all commits that are required to release the new version and optionally set the correct tags.
+
+Special considerations for major releases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Whenever you release a new major release you will be releasing more or less the master branch.
```

Comment:
```suggestion
Whenever you release a new major version you will be releasing more or less the master branch.
```
To avoid having 3 times release in the sentence.

Reply:
I thought about it. Actually you're releasing a major version. So I'll replace "major release" with "major version".